tensorflow_lite_support/python/task/processor/proto/segmentations_pb2.py

```diff
@@ -13,7 +13,207 @@
 # limitations under the License.
 """Segmentations protobuf."""
 
+import dataclasses
+from typing import Any, Tuple, List
+
+from tensorflow.tools.docs import doc_controls
 from tensorflow_lite_support.cc.task.vision.proto import segmentations_pb2
 
-Segmentation = segmentations_pb2.Segmentation
-SegmentationResult = segmentations_pb2.SegmentationResult
+_Segmentation = segmentations_pb2.Segmentation
+_ConfidenceMask = segmentations_pb2.Segmentation.ConfidenceMask
+_ColoredLabel = segmentations_pb2.Segmentation.ColoredLabel
+_SegmentationResult = segmentations_pb2.SegmentationResult
+
+
+@dataclasses.dataclass
+class ConfidenceMask:
+  """This is a flattened 2D-array in row major order. For each pixel, the
+    value indicates the prediction confidence usually in the [0, 1] range
+    where higher values represent a stronger confidence.
+    Ultimately this is model specific, and other range of values might be
+    used.
+
+  Attributes:
+    value: The value indicates the prediction confidence usually in the
+      range [0, 1].
+  """
+  value: float
```

Comment:
Can you make this a numpy 2D array instead of a flatten 2D float array? It will make handling the mask easier for the users. Please also update the docstring accordingly.

Reply:
@khanhlvg I've updated this in the dataclasses to use a 2d-array now.